### PR TITLE
fix str issue in models across Python versions

### PR DESCRIPTION
**Pull request by jtauber**
* adds test exhibiting problem identified in #17 
* updates mistakes in test matrix
* switches to `__str__` with `@python_2_unicode_compatible` in models

### CODE REVIEW

**Comment by jtauber** [comment]
Fixes #17

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ python:
   - "3.4"
   - "3.5"
 env:
-  - DJANGO=1.8
```
Comment (paltman) [inline .travis.yml:8]: @jtauber did you intend to drop Django 1.8 support?

**Review by paltman — COMMENTED** [review]
All looks good but I think we should keep 1.8 support and just add 1.9 support.

**Comment by jtauber** [comment]
@paltman yeah, I made a mistake on top of my earlier mistake :-) committed a fix

**Pull request merged**